package.json

```
@@ -2,17 +2,21 @@
   "name": "wdio-cucumberjs-json-reporter",
   "version": "2.0.4",
   "description": "A WDIO reporter that creates CucumberJS JSON files for WebdriverIO V5+",
-  "main": "./build/reporter.js",
+  "main": "./dist/lib/reporter.js",
```

so we can change this to 
```suggestion
  "main": "./dist/reporter.js",
```